src/main/java/brandkon/BrandController.java

```diff
@@ -0,0 +1,34 @@
+package brandkon;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.*;
+
+@RestController
+public class BrandController {
+
+    private BrandService brandService;
```

Comment:
`final` 까지 붙여 주시면 더 좋습니다
```suggestion
    private final BrandService brandService;
```